### DISCUSSION TITLE
Add a button to make it easy to report a package

### DISCUSSION THREAD

**Discussion by carols10cents** [rust-lang/crates.io#4343] [Feature Requests]
For whatever reason. I'm not sure how this should be implemented exactly yet, but we should definitely have something along these lines.

**Comment by asonix** [comment]
Has there been any movement on this front? This issue was opened over a year ago

**Comment by Turbo87** [comment]
@carols10cents where do you expect this button to report the package to? would it go to the support email of crates.io?

**Comment by carols10cents** [comment]
Yep!

**Reply by Turbo87** [reply]
do you think a simple `mailto:...` link would be sufficient for this?

**Reply by carols10cents** [reply]
I suppose, but I'd rather have something with a form that already fills out the name of the crate that the person is looking at and has a textarea prompting the person to explain why they're reporting it.

**Reply by Manishearth** [reply]
n.b. mailto links support `?subject=foo&body=bar`
